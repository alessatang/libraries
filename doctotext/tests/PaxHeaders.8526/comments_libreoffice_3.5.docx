30 atime=1389151481.246853891
30 ctime=1389151481.246853891
