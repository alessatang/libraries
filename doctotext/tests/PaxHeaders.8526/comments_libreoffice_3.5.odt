30 atime=1389151481.398853253
30 ctime=1389151481.398853253
