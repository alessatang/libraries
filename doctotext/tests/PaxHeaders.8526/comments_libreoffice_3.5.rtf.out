30 mtime=1379553523.870590382
30 atime=1389151481.407853215
30 ctime=1389151481.407853215
